src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -12433,7 +12433,7 @@ public void CS0698ERR_GenericDerivingFromAttribute01()
 {
 }
 ";
-            CreateCompilation(text).VerifyDiagnostics(
+            CreateCompilation(text, parseOptions: CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.CSharp7)).VerifyDiagnostics(
```

Comment:
Please duplicate such tests and use CSharp8 in the copy to illustrate the change.

Reply:
For the CSharp8 tests I need only to check that there are no error?
How\Where to test that the generated IL is correct?

Reply:
@jcouv I added 3 tests please review them
I'm working on the rest of the tests